WcaOnRails/app/models/locale.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+class Locale < SimpleDelegator
+  include ActionView::Helpers::TextHelper
+  SOME_CHARS = '[\s\S]*?'
```

Comment:
Oh right, I actually tested with the actual French locale and saw no difference in the output, but everything is "outdated" so this "test" was actually a pretty bad one...
I also thought the `?` would make the `.*` optional, and that it could match 0 or 1 char multiple time (which is why I thought I could remove it). Very messed up interpretation actually...